test/test_linint2.py

```diff
@@ -302,4 +302,4 @@ def test_linint2_non_contiguous_np(self):
                      xi=xi,
                      yi=yi_reverse[::-1])
         np.testing.assert_array_equal(fi[:, :, ::-1, :].values,
```

Comment:
`.values` can be removed from this line.

Reply:
Done.